Makefile

```diff
@@ -212,3 +212,10 @@ verify:
 	./hack/verify-boilerplate.sh
 	./hack/verify-clientset.sh
 	./hack/verify-bazel.sh
+
+## --------------------------------------
+## Others / Utilities
+## --------------------------------------
+
+diagrams:
```

Comment:
Is it diagrams, or figures? I'm fine with either, but simplest to use one :smile: